parl/core/paddle/tests/agent_base_test_paddle.py

```diff
@@ -27,19 +27,47 @@ def __init__(self):
         self.fc2 = nn.Linear(256, 128)
         self.fc3 = nn.Linear(128, 1)
 
+    def forward(self, obs):
+        out = self.fc1(obs)
+        out = self.fc2(out)
+        out = self.fc3(out)
+        return out
+
+
+class TestModelWithoutForward(parl.Model):
+    def __init__(self):
+        super(TestModelWithoutForward, self).__init__()
+        self.fc1 = nn.Linear(4, 256)
+        self.fc2 = nn.Linear(256, 128)
+        self.fc3 = nn.Linear(128, 1)
```

Comment:
please add `self.forward = None` for testing line 138 of parl/core/paddle/agent.py